template/en/default/account/auth/login.html.tmpl

```diff
@@ -84,6 +84,12 @@
   </form>
 </div>
 
+[% IF Param('oauth2_client_enabled') %]
+  [% INCLUDE account/auth/oauth2_login_button.html.tmpl
+     target => target
+  %]
+[% END %]
```

Comment:
Can you please include a screenshot of how the two login buttons are presented

Reply:
![image](https://user-images.githubusercontent.com/826315/94024456-71350800-fd85-11ea-94c1-2fe733b7e3aa.png)

![image](https://user-images.githubusercontent.com/826315/94024503-7db96080-fd85-11ea-81a0-d945ce75151e.png)

Reply:
It'll need to be styled to match GitHub's button (or vice versa, although having the service's icon is super useful).

Reply:
Probably should use something more standard such as 

https://lipis.github.io/bootstrap-social/

This is what Lando has currently